python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/mpls-rsvp.j2

```diff
@@ -0,0 +1,100 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - mpls interfaces #}
+{% if mpls.rsvp is arista.avd.defined %}
+
+### MPLS RSVP
+
+#### MPLS RSVP Summary
+
+| Setting | Value |
+| ------- | ----- |
+{%     if mpls.rsvp.refresh is arista.avd.defined %}
+{%         if mpls.rsvp.refresh.interval is arista.avd.defined %}
+| Refresh interval | {{ mpls.rsvp.refresh.interval }} |
+{%         endif %}
+{%         if mpls.rsvp.refresh.method is arista.avd.defined %}
+| Refresh method  | {{ mpls.rsvp.refresh.method }} |
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.hello.interval is arista.avd.defined and mpls.rsvp.hello.multiplier is arista.avd.defined %}
+| Hello interval | {{ mpls.rsvp.hello.interval }} |
+| Timeout multiplier | {{ mpls.rsvp.hello.multiplier }} |
+{%     endif %}
+{%     if mpls.rsvp.authentication is arista.avd.defined %}
+{%         set auth = mpls.rsvp.authentication %}
+| Authentication type | {{ auth.type | arista.avd.default('-') }} |
+| Authentication sequence-number window | {{ auth.sequence_number_window | arista.avd.default('-') }} |
+| Authentication active index | {{ auth.active_index | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.ip_access_group is arista.avd.defined or mpls.rsvp.ipv6_access_group is arista.avd.defined %}
+| IPv4 access-group | {{ mpls.rsvp.ip_access_group | arista.avd.default('-') }} |
+| IPv6 access-group | {{ mpls.rsvp.ipv6_access_group | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.srlg.strict is arista.avd.defined(true) %}
+| SRLG strict | enabled |
+{%     elif mpls.rsvp.srlg.enabled is arista.avd.defined(true) %}
+| SRLG | enabled |
+{%     endif %}
+{%     if mpls.rsvp.label_local_termination is arista.avd.defined %}
+| Label local-termination | {{ mpls.rsvp.label_local_termination }} |
+{%     endif %}
+{%     if mpls.rsvp.preemption_method.preemption is arista.avd.defined %}
+| Preemption method | {{ mpls.rsvp.preemption_method.preemption }} |
+{%         if mpls.rsvp.preemption_method.timer is arista.avd.defined %}
+| Preemption timer | {{ mpls.rsvp.preemption_method.timer }} |
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.mtu_signaling is arista.avd.defined %}
+| MTU signaling | {{ mpls.rsvp.mtu_signaling | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.fast_reroute is arista.avd.defined %}
+{%         set fast_reroute = mpls.rsvp.fast_reroute %}
+| Fast reroute mode | {{ fast_reroute.mode | arista.avd.default('-') }} |
+| Fast reroute reversion | {{ fast_reroute.reversion | arista.avd.default('-') }} |
+| Fast reroute  bypass tunnel optimization interval | {{ fast_reroute.bypass_tunnel_optimization_interval | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.hitless_restart is arista.avd.defined %}
+| Hitless restart | Active |
```

Comment:
The active will depend on enabled as True

Reply:
Fixed!